test/intl402/Segmenter/constructor/constructor/options-valid-combinations.js

```diff
@@ -25,35 +21,16 @@ combinations.push([
   undefined,
 ]);
 
-for (const lineBreakStyle of lineBreakStyleOptions) {
-  combinations.push([
-    { lineBreakStyle },
-    "grapheme",
-    undefined,
-  ]);
-}
-
 for (const granularity of granularityOptions) {
   combinations.push([
     { granularity },
     granularity,
-    granularity === "line" ? "normal" : undefined,
+    undefined,
```

Comment:
You can delete the third element of the array and then remove it in the for loop on line 32, where it is mapped to the local variable "lineBreakStyle".

Reply:
since lineBreakStyle was in the spec, I follow @littledan suggestion in other tests PR to keep it with a different expectation so see that property / value will actually have the right output instead of ignore and not checking it.

Reply:
I see.  Makes sense.